corehq/apps/data_interfaces/dispatcher.py

```diff
@@ -7,7 +7,7 @@
 from corehq.apps.reports.dispatcher import ReportDispatcher, datespan_default
 from corehq.apps.users.decorators import require_permission
 from corehq.apps.users.models import Permissions
-from custom.icds.view_utils import check_data_interfaces_blocked_for_domain
+from custom.icds_core.view_utils import check_data_interfaces_blocked_for_domain
```

Comment:
nit: `icds_cas` might be better than `icds_core` if you just want to create something different from `icds` and `icds_reports`